test/integration/multikueue/multikueue_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2023 The Kubernetes Authors.
```

Comment:
I feel this is independent of e2e tests, could you send a dedicated PR?

Reply:
I proposed to because the suites are independent, so one suite could merge earlier, once ready, but if you prefer to keep them together, I'm ok with that.

Reply:
it's simpler in one PR